hack/testdata/levee-config.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
I would recommend a less generic name for this subdir, and giving it an OWNERS file with focused approvers so you don't have to keep hitting /hack approvers

Reply:
I've added a `testdata` subdirectory to hold the configuration and related OWNERS file.  Is that how you meant your comment for a less generic subdir?  `testdata` itself is already home to 100+ files, so I didn't figure you meant that when you said "this subdir."  Alternatively, I could just update the `testdata` OWNERS with a filter on this config, if that seems better organized.

It's on the SIG Security agenda this afternoon to discuss taking ownership of this tool, but I've gone ahead and put `sig-security` as the tentative OWNERS for the config.

Reply:
Filters don't work well for approvers.

I meant instead of `/hack/testdata` which sounds like it should be owned by all kinds of different people, use something named after the tool you're owning.  `/hack/testdata/levee` is fine.  I would also have been OK with something like `/hack/levee`